### ISSUE TITLE
"online login is currently unavaliable" in Origin.

### ISSUE THREAD

**Issue by HasanQZ** [lutris/lutris#3109]
When attempting to login to Origin, I get the message "online login is currently unavaliable".
When trying to login, it says "You must be online when logging in for the first time".

**Steps to reproduce**
- Install origin or any game that runs under Origin on lutris.

The terminal output of `lutris -d`:
```
Unable to load locale dir, translations won't work.
2020-08-23 11:25:44,168: MAME XML generation launched in the background, not returning anything this time
2020-08-23 11:25:44,168: Getting full game list from MAME...
2020-08-23 11:25:44,169: MAME isn't installed, can't retrieve systems list.
2020-08-23 11:25:44,259: MAME XML written
```

**System information**
OS: Arch Linux
DE: GNOME 3

Network: ethernet, no vpn or proxy
GPU: Nvidia GTX1060 3gb
CPU: AMD fx-6300
RAM: 8GB

Game logs:

[lutrislogs.txt](https://github.com/lutris/lutris/files/5114308/lutrislogs.txt)

Origin sign in works everywhere else.

**Comment by byte1024** [comment]
This might be a wine or script issue. In any case, I just tested and get in to offline mode successfully which is how I had it but it refused to go online and said there was a mandatory update required for that. Tried getting the update, and after some attempts, Origin completely broke on me.

Origin is a bit messy with Wine and various issues so it will probably be sorted in time. I didn't yet dig into my own log or try a clean install, etc so if I get it working, will update.

**Comment by byte1024** [comment]
After almost giving up, I did yet another new install of Origin from lutris.net script (maybe 5th try but 1st completely default) and it signed in and got online.

There's a message that on August 18th origin and all games will close and restart for an automatic update.

But, the install shows the latest version is installed for this one. So, try again with installing it. I also recommend a new directory along with all default install settings (don't change a single thing).

**Comment by HasanQZ** [comment]
I found the issue! After hours of troubleshooting, I realized that there were a _lot_ of "optional" dependancies. I installed **ALL** of them, and then i reinstalled Lutris, Wine, Winetricks, PlayOnLinux, etc. Then i reinstalled everything again including all the optional dependencies, and now i can log into Origin as normal. I used the default install settings, only changing the directory, and everythinig worked. Thanks for your help!
